### ISSUE TITLE
KakaoAccountResult returning null except for userId

### ISSUE THREAD

**Issue by charkala** [JosephNK/flutter_kakao_login#15]
First of all thank you @JosephNK for creating this, I find it much better than the other package out there since it's much easier to use 👍 

My issue is that `final KakaoLoginResult result = await kakaoSignIn.getUserMe();` returns `null` except for `userId`.

When I debugged `kakaoSignIn.getUserMe()` method, the result seems to be there.

```
login_page.dart

final KakaoLoginResult result = await kakaoSignIn.getUserMe();
print('userName: ${result.account.userNickname}');

flutter: userNickname: null
```

```
flutter_kakao_login.dart

// Get UserMe Method
Future<KakaoLoginResult> getUserMe() async {
    final Map<dynamic, dynamic> result =
        await _channel.invokeMethod('getUserMe');

        print('debug: ${result.toString()}');

        return _delayedToResult(
            new KakaoLoginResult._(result.cast<String, dynamic>()));
 }

flutter: debug: {status: loggedIn, userID: xxxxxxxxxx, userNickname: xxxxxxxx}
```

Not sure why it won't return the value.

**Comment by JosephNK** [comment]
Hi @charkala.
Do you get a nickname?
userNickname and userID is null?
What is a test device? iOS? Android?

**Comment by charkala** [comment]
@JosephNK Hello.
`await kakaoSignIn.getUserMe()` returns only `userID: xxxxxxxxx`, all other fields are `null`
but when I print from inside the package (see my `print()` line) it returns both `userID` and `userNickname`.

For my Kakao Dev Console, I've enabled `profile (nickname/profile photo)` and `account (email)`.
I've also setup the SDK from their guide with no problem.

I have tested on both devices, Android (Samsung) and iOS (iPhone) and all same results.
